qiita_db/portal.py

```diff
@@ -0,0 +1,53 @@
+from .sql_connection import SQLConnectionHandler
+from .util import convert_to_id
+from qiita_core.qiita_settings import qiita_config
+
+# Only make functions available for main portal
+if qiita_config.portal == "QIITA":
+    def get_studies_by_portal(portal):
```

Comment:
I think it's fine either way, though we certainly have been using the
design that @josenavas suggests all over the place.

On (Jun-22-15| 8:56), Joshua Shorenstein wrote:
> > @@ -0,0 +1,53 @@
> > +from .sql_connection import SQLConnectionHandler
> > +from .util import convert_to_id
> > +from qiita_core.qiita_settings import qiita_config
> > +
> > +# Only make functions available for main portal
> > +if qiita_config.portal == "QIITA":
> > +    def get_studies_by_portal(portal):
> 
> As we previously discussed, these are essentially helper functions for the `qiita-env` command line commands so to me it makes no sense to put them into an object. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1256/files#r32950236